spec/views/devise/mailer/confirmation_instructions.html.slim_spec.rb

```diff
@@ -11,10 +11,7 @@
   end
 
   it 'includes the support link' do
-    assign(:resource, build_stubbed(:user, confirmed_at: Time.zone.now))
-    render
-
-    expect(rendered).to have_link(Figaro.env.support_url, href: Figaro.env.support_url)
+    to include Figaro.env.support_url
```

Comment:
@monfresh This test is failing because the link to the support url has been moved into the user_mailer layout.  This test seems to only look at the rendered content of the actual confirmation_instructions.html.slim file which no longer includes the support url.  I used last_email in order to see the entire email, but that may not be the correct way to do it.